### PR TITLE
GSoC][TMVA] Add PyTorch Tutorials

### PR DESCRIPTION
**Pull request by omazapa**
Hello,

In this PR I am submitting the code for tutorials  from my GSoC student @AnirudhDagar with some fixes.

1) Fixed TorchScript problem
2) Fixed tutorial with wrong dimesions while evaluating the loss function.

I closed  this PR https://github.com/root-project/root/pull/6273

Best Omar

### CODE REVIEW

**Comment by bellenot** [comment]
Hi @omazapa ! Can you fork also roottest? That will prevent build failure (timeout) on Windows. Thanks!

**Comment by omazapa** [comment]
Hi @bellenot!

Just doing the fork? should I do somehting aditional in the fork?
the fork is done 
Cheers

File: tutorials/tmva/TMVA_CNN_Classification.C [hunk 1]
```diff
@@ -107,14 +107,14 @@ void MakeImagesTree(int n, int nh, int nw)
    f.Close();
 }
 
-void TMVA_CNN_Classification(std::vector<bool> opt = {1, 1, 1, 1})
+void TMVA_CNN_Classification(std::vector<bool> opt = {0, 0, 0, 0, 1})
```
Comment (lmoneta) [inline tutorials/tmva/TMVA_CNN_Classification.C:110]: I would add here {1,1,1,1,0}   
switch off pytorch by default since nobody has it installed

Comment (lmoneta) [reply tutorials/tmva/TMVA_CNN_Classification.C:110]: And add also off at line 147

**Comment by bellenot** [comment]
> Just doing the fork? should I do somehting aditional in the fork?

No, the fork is enough, thanks! 👍 😃

**Review by lmoneta — APPROVED** [review]

**Comment by lmoneta** [comment]
I think we can merge the PR. 
Maybe Omar, it is better to merge all commits of Anirudh in a single commit  and the others in another one before merging it to have a simpler history

**Comment by lmoneta** [comment]
@phsft-bot build!
